RocketChat/Rocket.Chat.ReactNative
# [FIX] Update Loading logo

## Proposed changes

#2580 missed Loading component

## Issue(s)	


## How to test or reproduce

- Run any action that triggers Loading component (like change status message)
- New logo should be shown

## Screenshots
<img src="https://user-images.githubusercontent.com/804994/99694870-a26f4300-2a6b-11eb-9792-65e383016f17.png" width="350" />


## Types of changes




## Checklist



## Further comments
